constants.py

```diff
@@ -184,9 +184,7 @@
 kBackLeftModuleName = "back_left"
 kBackRightModuleName = "back_right"
 
-kLimelightMountingOffset = Translation2d(
-    (kRobotLength / 2) - (4.125 * kMetersPerInch), 0.0
-)
+kLimelightMountingOffset = Translation2d(0.0, 0.0)
```

Comment:
I assume this is an implementation thing? Is limelight the center of odometry or something like that?

Reply:
limelight is in the middle of the robot